build.py

```diff
@@ -64,21 +64,28 @@
 # different versions are used then one backend or the other will
 # incorrectly load the other version of the openvino libraries.
 #
+# Adding another openvino version. Take note of the above issue
+# when loading mulitple backends with different openvino versions.
+# To use openvino_21_4, the model configuration should specify
+# the "openvino_21_4" in the "backend" field.
+#
 TRITON_VERSION_MAP = {
     '2.18.0dev': (
         '22.01dev',  # triton container
         '21.11',  # upstream container
         '1.10.0', # ORT
         '2021.2.200',  # ORT OpenVINO
         '2021.2',  # Standalone OpenVINO
```

Comment:
Can we generalize this entry to be a list of OpenVINO versions. Or since we need to support both building from source and installing prebuilt perhaps we can express like (('2021.2', None /* build from source */), ('2021.4', '2021.4.582'))
Also update the documentation above to describe. Also have the first entry in the list be the default so that 'openvino' backend refers to the default.